remme/clients/basic.py

```diff
@@ -322,8 +332,7 @@ def _send_transaction(self, method, data_pb, addresses_input, addresses_output):
         addresses_input_output = list(set(addresses_input_output))
         # forward transaction to test helper
         if self.test_helper:
-            self.test_helper.send_transaction(method, data_pb, addresses_input_output)
-            return
+            return self.test_helper.send_transaction(method, data_pb, addresses_input_output)
```

Comment:
WTF is this test_helper? Once again: code being ran in tests should be the same as in production.

Reply:
This is a client's code (not TP) to support unit testing

Reply:
Once again: the behavior should not depend on whether we are running on a test or production environment. This is the basic principle of unit testing.

Reply:
Agreed with @eugene-babichenko, this must not be present here. For testing purpose, u need to create directly in test helpers and not reuse the bl function with different behavior.